### PR TITLE
Add multi-select documentation

### PR DESCRIPTION
**Pull request by sebko**
Trello URL: https://trello.com/c/NFIvpZSN/4553-1-multi-select-add-documentation-for-multi-select-builder-input-type

### CODE REVIEW

File: content/docs/200-inputs/600-multi-select.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Multi-select
+excerpt: Select from a large list of multi-select options.
+---
+
+Add a _multi-select_ input to your app to allow users select from a large list of multi-select options.
```
Comment (jschr) [inline content/docs/200-inputs/600-multi-select.mdx:6]:
```suggestion
Add a _multi-select_ input to your app to allow users select multiple values from a list of options.
```

Just a suggestion to remove the "multi-select" repetition. Also maybe drop the word "large" here. I used "large" in the Select description to differentiate it from Button Group but in this case there's only the Multi-select input when you want to select multiple values.

File: content/docs/200-inputs/600-multi-select.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Options
+
+Add options to your _multi-select_ input and set a default option. The input ID in the URL template for your _multi-select_ input is replaced with the selected option’s values.
```
Comment (jschr) [inline content/docs/200-inputs/600-multi-select.mdx:10]: I think we probably need to talk about how these options are passed in via the query string and how to decode them here too.

File: content/docs/200-inputs/400-select.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Select
-excerpt: Select from a large lists of options.
+excerpt: Select from a large list of single-select options.
```
Comment (jschr) [inline content/docs/200-inputs/400-select.mdx:3]:
```suggestion
excerpt: Select a single value from a large list options.
```

Curious if you think this reads a bit better?

File: content/docs/200-inputs/600-multi-select.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+---
+title: Multi-select
+excerpt: Select from a large list of multi-select options.
```
Comment (jschr) [inline content/docs/200-inputs/600-multi-select.mdx:3]:
```suggestion
excerpt: Select multiple values from a list of options.
```

File: content/docs/200-inputs/600-multi-select.mdx (hunk 2 above)
Comment (sebko) [reply content/docs/200-inputs/600-multi-select.mdx:10]: Good call! See below.

https://github.com/mirainc/developer-docs/pull/42/files#diff-1b6bce6314f8c4e245fb5677670f12e1991bba222d644d0b1ae9f3a867000c0dR12-R27

**Review by jschr — APPROVED** [review]

**Pull request merged**